client/blocks/comments/comment-actions.scss

```diff
@@ -4,59 +4,62 @@
 	font-size: $font-body-small;
 	list-style: none;
 	margin-top: -6px;
-
-	button {
+	
+	button.button {
 		display: inline-block;
-	color: var( --color-text-subtle );
+		color: var( --color-text-subtle );
```

Comment:
This style conflicts with the one that gives the "Read more" button the primary color:
```css
.comments__comment-actions .comments__comment-actions-read-more {
	color: var( --color-primary );
}
```

That causes the coloring issue that @tyxla reports in https://github.com/Automattic/wp-calypso/pull/45344#pullrequestreview-481633015